### ISSUE TITLE
[v4] custom-select in an input-group, with a label

### ISSUE THREAD

**Issue by jcoyne** [twbs/bootstrap#25207]
The `.custom-select` in an input group sets the `border-*-left-radius` to zero if it is not the first child in an `input-group` (`.input-group .custom-select:not(:first-child)`).  This assumption is broken if we want to add a label to this select for screen reader users.  e.g.:

```html
<div class="input-group">
      <label for="my-select" class="sr-only">Screen reader's label</label>
      <select class="custom-select" id="my-select">...</select>
</div>
```

This was tested in 4.0.0.beta3
Screenshot:
<img width="748" alt="screen shot 2018-01-05 at 9 57 17 am" src="https://user-images.githubusercontent.com/92044/34616796-defe9936-f1fe-11e7-8655-1f2613763687.png">

**Comment by jcoyne** [comment]
Perhaps the documentation stating: "We do not support multiple form-controls in a single input group and <label>s must come outside the input group." takes care of this.

**Comment by ysds** [comment]
Fixed in #25173 :)
